.azure-pipelines.BuildMatrix.yml

```diff
@@ -0,0 +1,138 @@
+jobs:
+- job: Build
+
+  pool:
+    vmImage: 'windows-2022'
+
+  strategy:
+    matrix:
+      #$json="{'job1': {'Work': '**\AdvancedXBind.sln'}, 'job2': {'Work': '**\AndroidCustomCamera.sln'}}"
+
+    # AdvancedXBind:
+    #   sampleName: '**\AdvancedXBind.sln'
+
```

Comment:
It could make sense to use this: https://stackoverflow.com/a/69345058

to generate the matrix based on the folders.